### ISSUE TITLE
debugging tests can't find any source code, even the test that's running

### ISSUE THREAD

**Issue by dabraham02124** [cbeust/testng-eclipse#490]
### Problem Statement
When I debug a test, it can't find the code for the project, even the code that is the test that I'm running.

This happened when I updated my testng plugin recently (I don't know what I had before I updated, but several months old).  Since then I've tried updating to the latest beta of the plugin (BTW, the run/debug buttons in the code are neat, thank you).  Still there.

Then I tried uninstalling, then reinstalling the latest release, still there.

I can get around it _for a specific debug target_, by clicking on the "Edit Source Lookup Path..." button, then "Add...", then "Java Project", click the box next to the name of the project that my test is in, then "OK", then "OK".

It continues to work if I debug that specific target again.  But if I debug _anything_ else, even another test in the same file, I have to go through the rigamarole again.

### Any relate message in "Error Log" view
> "Windows -> Show View -> Others -> Error Log" 

Since I reinstalled the last time, when I filter on "test" the only thing that I see is "[TestNGLaunchConfigurationDelegate] Launching:"

### The Dependency Management tool for your project
- [X] Maven

### Operating System
- [X] OSX

### Eclipse
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Comment by missedone** [comment]
thanks @dabraham02124 for reporting this, i can reproduce this on Eclipse 2020-03 as well. probably this new Eclipse version changed something internally.
haven't look in depth yet.

@gayanper, could you like to have a try to fix this issue?

Thanks
Nick

**Comment by gayanper** [comment]
Please switch the m2e snapshots this should be fixed there. This is a m2e bug.

**Comment by dabraham02124** [comment]
@gayanper thank you very much, yes, that did it.

For the next person who bumps into this, I updated m2e from 1.15.0.20200310-1832 to 1.16.0.20200416-1943 by installing from [here](https://download.eclipse.org/technology/m2e/snapshots/1.16.0/latest/).

What I did (which may be overly paranoid, but worked) was:
1. uninstall the testng plugin & restart eclipse
1. update m2e with the link above & restart eclipse
1. install the testng plugin & restart eclipse
